src/core/Action.ts

```diff
@@ -67,7 +68,7 @@ export default class Action<
       return;
     }
 
-    const actionGoalId = `send_action_goal:${this.name}:${(++this.ros.idCounter).toString()}`;
+    const actionGoalId = `send_action_goal:${this.name}:${uuidv4()}`;
```

Comment:
Can we replace other places to also use template literals instead of string concatenation, for consistency?

Reply:
done via an eslint rule